operators/src/plot/box_plot.rs

```diff
@@ -49,10 +49,13 @@ pub struct BoxPlotParams {
 impl PlotOperator for BoxPlot {
     async fn _initialize(
         self: Box<Self>,
+        path: WorkflowOperatorPath,
         context: &dyn ExecutionContext,
     ) -> Result<Box<dyn InitializedPlotOperator>> {
-        match self.sources.source {
-            MultiRasterOrVectorOperator::Raster(raster_sources) => {
+        let initialized_sources = self.sources.initialize_sources(path, context).await?;
```

Comment:
can yo udo the initialization after checking the number of inputs, like before?

Reply:
this is not so easy because the match uses the inner enum. We could remove the `InitializedSources` trait. And then generate initialized sources structs by calling individual constructors. There we could pass validators.

Reply:
[7136c37](https://github.com/geo-engine/geoengine/pull/780/commits/7136c37923939b32082fb9d1c81bf81cb0785c08)